docs/README.md

```diff
@@ -1,31 +1,26 @@
-# InterBTC & Kintsugi Documentation
+# Interlay & Kintsugi Documentation
+### Invest, earn and pay with BTC on any blockchain. Radically open and secured by insurance.
 
-## Radically Open Bitcoin for Decentralized Finance
 
-**interBTC** is Bitcoin on any blockchain. A 1:1 Bitcoin-backed asset, fully collateralized, interoperable, and censorship-resistant. interBTC will be hosted as a Polkadot parachain and connected to Cosmos, Ethereum and other major DeFi networks.
+### Introduction
 
-**Kintsugi** is interBTC's canary network, an experiment with real economic value deployed on Kusama. Kintsugi and interBTC share the same code base - with the difference that Kintsugi focusses strictly on innovation and will always be ahead in terms of features.
+Interlay is a decentralized network dedicated to connecting crypto-*currencies* like Bitcoin with DeFi *platforms* like Polkadot and Ethereum. The Interlay network is hosted as a Polkadot parachain and will be connected to Cosmos, Ethereum and other major DeFi networks.
+[Read more about Interlay's vision of blockchain interoperability](https://medium.com/interlay/the-future-of-bridging-assets-837998115f6b).
 
-![Cryptocurrency-backed Assets](_assets/img/CbA.jpg)
+**interBTC**, Interlay's flagshipt product, is Bitcoin on any blockchain. A 1:1 Bitcoin-backed asset, fully collateralized, interoperable, and censorship-resistant. 
 
-### Issue, Trade, and Redeem
+**Kintsugi** is Interlay's canary network, an experiment with real economic value deployed on Kusama. Kintsugi and Interlay share the same code base - with the difference that Kintsugi focusses strictly on innovation and will always be ahead in terms of features. [Read more about Interlay's dual network model](https://medium.com/interlay/the-interlay-parachain-is-coming-to-polkadot-552a57ff8d1b).
 
-The life-cycle of interBTC follows the three main protocols:
 
-- **Issue:** Lock your BTC with a Vault. Pick one, or run your own. Your BTC is always safe and insured by Vault collateral.
-- **Mint:** Get interBTC at a 1:1 ration to your locked BTC.
-- **BTC DeFi:** Earn on your Bitcoin. Use interBTC as collateral, for lending, yield farming and more. On Polkadot, Kusama, Cosmos, Ethereum and other major DeFi platforms.
-- **Redeem:** Redeem interBTC for actual BTC on Bitcoin - trustless and anytime you want.
+### Docs vs Specification vs Code
```

Comment:
Instead of a paragraph about the documentation itself, I think a comparison of the different wrapped Bitcoin tokens or some form of competitor analysis should come here. Like why are we the best product.

I would remove this part of the documentation, or rather list is as something like:

```
## Further Reading
1. Specification
2. Code